charts/leonardo/templates/zombieMonitorConfigMap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: leonardo-zombie-monitor-cm
+  labels:
+{{ include "leonardo.labels" . | indent 4 }}
+data:
+  application.conf: |
+    leonardo-pubsub.google-project = {{ .Values.cronjob.googleProject }}
```

Comment:
`googleProject` should probably move from `cronjob` to `commonCronjob` since it's referenced in both resourceValidator and zombieMonitor yaml's but not a blocker.

Reply:
it's not reallly needed for zombie monitor...it's only there because Config.scala was put in `core`